Electrical bootcamp

scholvat opened the pull request:
Changed embedded bootcamp to electrical bootcamp. Pretty much is a basic training in DipTrace.

scholvat committed:
Updated Electrical Bootcamp

Renamed bootcamp to electrical and changed it to tracking antenna
project. Now it is pretty much a basic Diptrace project so they get used
to using that program.

scholvat committed:
changed name in mkdocs file

changed name from embedded to electrical

sergei1152 commented on docs/bootcamp/electrical.md:
```diff
@@ -0,0 +1,65 @@
+http://diptrace.com/download-diptrace/
+
+If you need help with using the CAD software, here are some resources:
+Diptrace Tutorial: http://diptrace.com/support/tutorials/
```
A markdown link is in the format of `[text of link](actual link)`. Start a server with `mkdocs serve` to verify the results

sergei1152 commented on docs/bootcamp/electrical.md:
```diff
@@ -0,0 +1,65 @@
+﻿# WARG Electrical Bootcamp
```
Add a blank space on the first line so the header parses correctly in markdown

scholvat committed:
Added links, changed datalink

When I tried to use mkdocs serve, datalink threw some errors. I made
some changes to the table since that's where the error was coming from,
though I'm not sure if these are actual issues or if I just don't have
mkdocs setup correctly.

scholvat committed:
Minor spacing change

Minor spacing change
